core/method_bind.cpp

```diff
@@ -34,20 +34,22 @@
 
 #include "method_bind.h"
 
-#ifdef DEBUG_METHODS_ENABLED
 PropertyInfo MethodBind::get_argument_info(int p_argument) const {
 	ERR_FAIL_INDEX_V(p_argument, get_argument_count(), PropertyInfo());
 
 	PropertyInfo info = _gen_argument_type_info(p_argument);
+#ifdef DEBUG_METHODS_ENABLED
 	info.name = p_argument < arg_names.size() ? String(arg_names[p_argument]) : String("arg" + itos(p_argument));
+#else
+	info.name = String("arg" + itos(p_argument));
+#endif
```

Comment:
I wonder if we could get away with not setting any name in the `#else` case. We care about the return type, but shouldn't need the `arg[0-9]` names. I guess that might have been what @neikeq referred to in the original PR.

Reply:
It's not what neikeq referred to, but I removed this since it's probably not needed.